test/org/zaproxy/zap/extension/websocket/utility/Utf8UtilUnitTest.java

```diff
@@ -2,8 +2,7 @@
 
 import org.junit.Test;
 
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.*;

```

Comment:
Please use specific imports.

Reply:
Oh sorry I should be more careful. My IDE changed that.

Reply:
No worries, it happens. I think that's a default for IntelliJ (is that what you're using?).

Reply:
Yeap, I going to configure it